modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -81,7 +83,21 @@ private String _fixImports(JavaClass javaClass, String content)
 		if (!newVariables.isEmpty()) {
 			newJavaClassContent = StringUtil.replace(
 				javaClassContent, ArrayUtil.toStringArray(variables),
-				ArrayUtil.toStringArray(newVariables));
+				ArrayUtil.toStringArray(newVariables), true);
+
+			for (int i = 0; i < variables.size(); i++) {
+				String regex = StringBundler.concat(
+					"\\b([_a-z]\\w*)", variables.get(i), "\\b");
+
+				Pattern pattern = Pattern.compile(regex);
+
+				Matcher matcher = pattern.matcher(content);
+
+				if (matcher.find()) {
+					newJavaClassContent = newJavaClassContent.replaceAll(
+						regex, matcher.group(1) + newVariables.get(i));
+				}
+			}
```

Comment:
@TamyrisTorres there is a reason for using for instead of foreach?

Can we do this?

```
Pattern pattern = Pattern.compile(StringBundler.concat("\\b([_a-z]\\w*)", variables.get(i), "\\b"));

```

Or the SF requested this variable?